src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/DefineContentStep.tsx

```diff
@@ -71,6 +71,7 @@ export default function DefineContentStep() {
           setDisabled
         >
           <DropdownMenu
+	    ouiaId='restrict_to_architecture'
```

Comment:
i think you may be able to do: `dropDownProps={{ouiaId: 'restrict_to_architecture'}}`, but maybe @adewar has a better way to add this that's more consistent with the other components?

Reply:
Yup, unsurprisingly @xbhouse  is correct.

Reply:
Thank you. I'll try add a few more.

Reply:
@Andrewgdewar not working yet

Reply:
Working now with 
menuToggleProps={{ 'data-ouia-component-id': 'restrict_to_os_version' }}
and 
menuToggleProps?: Partial<MenuToggleProps | unknown>;